msl_expressions/autogenerated/src/Plans/Behaviours/PositionReceiver.cpp

```diff
@@ -1,111 +1,105 @@
-            query->additionalPoints = additionalPoints;
-            query->velocityMode = msl::MovementQuery::Velocity::FAST;
-            mc = rm.moveToPoint(query);
-        }
-        else
-        { // still enough time to position ...
-//            mc = msl::RobotMovement::moveToPointCarefully(egoTarget, egoBallPos, slowCatchRadius, additionalPoints);
-            query->egoDestinationPoint = egoTarget;
-            query->egoAlignPoint = egoBallPos;
-            query->snapDistance = slowCatchRadius;
-            query->additionalPoints = additionalPoints;
-            query->velocityMode = msl::MovementQuery::Velocity::DEFAULT;
-            mc = rm.moveToPoint(query);
-        }
+    query->egoDestinationPoint = egoTarget;
+    query->egoAlignPoint = egoBallPos;
+    query->additionalPoints = additionalPoints;
+    mc = rm.moveToPoint(query);
 
-        // if we reach the point and are aligned, the behavior is successful
-        if (mc.motion.translation == 0 && fabs(egoBallPos->rotate(M_PI)->angleTo()) < (M_PI / 180) * alignTolerance)
-        {
-            this->setSuccess(true);
-        }
-        if (!std::isnan(mc.motion.translation))
-        {
-            send(mc);
-        }
-        else
-        {
-            cout << "Motion command is NaN!" << endl;
-        }
-        /*PROTECTED REGION END*/
+    // if we reach the point and are aligned, the behavior is successful
+    if (mc.motion.translation == 0 && fabs(egoBallPos->rotate(M_PI)->angleTo()) < (M_PI / 180) * alignTolerance)
+    {
+        this->setSuccess(true);
     }
-    void PositionReceiver::initialiseParameters()
+    if (!std::isnan(mc.motion.translation))
     {
-        /*PROTECTED REGION ID(initialiseParameters1439379316897) ENABLED START*/ //Add additional options here
-        readConfigParameters();
-        /*PROTECTED REGION END*/
+        send(mc);
     }
-    /*PROTECTED REGION ID(methods1439379316897) ENABLED START*/ //Add additional methods here
-    void PositionReceiver::readConfigParameters()
+    else
     {
-        supplementary::SystemConfig* sc = supplementary::SystemConfig::getInstance();
-        fastCatchRadius = (*sc)["Drive"]->get<double>("Drive.Fast.CatchRadius", NULL);
-        slowCatchRadius = (*sc)["Drive"]->get<double>("Drive.Carefully.CatchRadius", NULL);
-        alignTolerance = (*sc)["Drive"]->get<double>("Drive.Default.AlignTolerance", NULL);
-        ballDistanceRec = (*sc)["Drive"]->get<double>("Drive.KickOff.BallDistRec", NULL);
+        cout << "Motion command is NaN!" << endl;
     }
+    /*PROTECTED REGION END*/
+}
+void PositionReceiver::initialiseParameters()
+{
+    /*PROTECTED REGION ID(initialiseParameters1439379316897) ENABLED START*/ // Add additional options here
+    readConfigParameters();
+    /*PROTECTED REGION END*/
+}
+/*PROTECTED REGION ID(methods1439379316897) ENABLED START*/ // Add additional methods here
+void PositionReceiver::readConfigParameters()
```

Comment:
This method can probably removed because its only used in initialiseParameters anyway